tests/tools/TestApp/Inc/mcm.h

```diff
@@ -1,19 +1,18 @@
 #ifndef _MCM_H_
 #define _MCM_H_
-#include "mcm_mock.h"
 
-#define DUMMY_LEN 1
-typedef struct mcm_ts{
-    MeshConnection *connection;
-    MeshClient *client;
-}mcm_ts;
+#include <stdio.h>
+#include "mcm_dp.h"
+#include "mesh_dp.h"
 
+#define BYTE_SIZE 1
+#define FIRST_INDEX 0
 
-int mcm_init_client(mcm_ts* mcm, const char* cfg);
-int mcm_create_tx_connection(mcm_ts* mcm, const char* cfg);
-int mcm_create_rx_connection(mcm_ts* mcm, const char* cfg);
-int mcm_send_video_frame(mcm_ts* mcm, const char* frame, int frame_len);
-int mcm_receive_video_frames(mcm_ts* mcm);
-int file_to_buffer(FILE* fp, MeshBuffer* buf, int frame_size);
-
+int mcm_init_client(MeshConnection **connection, MeshClient *client, const char* cfg);
```

Comment:
```suggestion
int mcm_init_client(MeshConnection **connection, MeshClient *client, const char *cfg);
```
Make sure you have the same code style everywhere. It'll improve readability of the code and will make other developer's life easier :-)

Reply:
run clang-formater on all files